# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/xcdroast/no-version-check.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./init.c.orig	Tue Jul 17 21:39:11 2001
+++ ./init.c	Mon Dec 31 10:34:01 2001
@@ -566,7 +566,7 @@
 	debug = 0;
 	oldfontcode = 0;
 	ignorescanbus = 0;
-	noversioncheck = 0;
+	noversioncheck = 1; /* Rene for dRock: the version check sucks! */
 	alt_dev_string = NULL;
 	strncpy(configdir, CONFIGDIR, MAXLINE);
 	strncpy(rootconfig, ROOTCONFIG, MAXLINE);
